### ISSUE TITLE
Error on new helm deployment

### ISSUE THREAD

**Issue by atedsimple** [helm/helm-mapkubeapis#83]
Firstly I love this plugin, it has saved us so much pain

One thing though, when we start a new service that has never been deployed to AKS, it generates and error. It doesn't really make sense for this to be an error situation though

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/50109927/202685964-c20a6d82-45db-40f7-a762-65321ccd328d.png">

**Comment by hickeyma** [comment]
@atedsimple Its hard to know what is causing this. Would you be able to troubleshoot this further and provide more details?

**Comment by atedsimple** [comment]
Well it happens only when the service isn't already deployed to the cluster

**Comment by hickeyma** [comment]
@atedsimple What "service" are you referring to? Is this some service that you deploy using Helm?

**Comment by atedsimple** [comment]
Yeah. We create a service, package it up with Helm and deploy it to our AKS cluster. If the service is already deployed to our AKS cluster then Helm will upgrade it. However the upgrade will only work if there is a valid rollback path, so the current Helm deployment in the AKS cluster needs to be valid. Normally it is, however, if the AKS cluster has been upgraded then sometimes there is a breaking change to one of the AKS System API's so current Helm deployments become invalid. The actual services running within the cluster are fine, but the files used by Helm to deploy that version are not so any Helm upgrade of that service will fail as there is no valid rollback path. The mapkubeapis plugin is great as it detects and fixes these issues that are present in the current Helm deployment files so that the Helm upgrade that we are trying to run will work. However, if there is no current deployment present in the cluster then mapkubeapis generates an error. It would be better if it didn't generate an error as this will fail a DevOps Pipeline unless, of course, you set the Ignore Errors flag but then you get an unnecessary warning icon.

**Comment by hickeyma** [comment]
I can see your use case alright. Not really sure the best approach to do this as it handled by the error handling and cobra cli.

Other approach if you check for the release first before calling the tool?
